Lib/compositor/font.py

```diff
@@ -64,7 +64,7 @@ def loadInfo(self):
             platEncID = nameRecord.platEncID
             langID = nameRecord.langID
             text = nameRecord.string
-            nameIDs[nameID, platformID, platEncID, langID] = text
+            nameIDs[nameID, platformID, platEncID, langID] = text.toStr()
```

Comment:
wait, `toStr` is a method of fonttools' `NameRecord` class, not of built-in str or bytes objects.
you should call it in the line immediately before,  `text = nameRecord.toStr()` or `text = str(nameRecord)`